doc/sphinx/azhelpgen/doc_source_map.json

```diff
@@ -44,5 +44,6 @@
     "tag": "src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py",
     "vm": "src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py",
     "vmss": "src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py",
-    "servicefabric": "src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_help.py"
+    "servicefabric": "src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_help.py",
+	"eventhubs": "src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_help.py"
```

Comment:
Keep the indentation the same as with the others.
It looks not aligned here.

Reply:
aligned